packaging/_parser.py

```diff
@@ -0,0 +1,155 @@
+from typing import Any, List, Tuple
```

Comment:
I'd appreciate it if we can get rid of as many of the `Any`s here as feasible. Not a blocker, obviously.

Reply:
I know that @hrnciar is working on redoing the typing annotations not to use Anys. Most of the typing annotation comments from this review will be resolved by his WIP changes.

Reply:
And I imagine our mypy lint job is complaining about them as well. :)